src/components/NewLocationPage/CompareLocationTabs/CompareLocationTabs.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { ButtonGroup, Button } from './CompareLocationTabs.style';
+import {
+  HomepageLocationScope,
+  homepageLabelMap,
+  GeoScopeFilter,
+} from 'common/utils/compare';
+
+function getGeoScopeCopy(
+  geoScope: GeoScopeFilter | HomepageLocationScope,
+  stateId?: string,
+) {
+  switch (geoScope) {
+    case GeoScopeFilter.NEARBY:
+      return 'Nearby';
+    case GeoScopeFilter.STATE:
+      return stateId;
+    case GeoScopeFilter.COUNTRY:
+      return 'USA';
+    default:
+      return null;
+  }
+}
```

Comment:
Not the biggest fan of this implementation 🤔
- Currently `HomepageLocationScope` is used as a possible type in the parameter, although there shouldn't be a case when an object of this type will be passed to this function. The only reason it's currently there is to prevent an error.
- Straight up returning a parameter as a return value seems inefficient. `StateId` is merely there as I try to be explicit about covering all possible types of `GeoScopeFilter`.
- Default case should never be reached in this case... but again only there for the purpose of being explicit.
IMO, this is just all around not the best function I came up with and I would love to talk through a potentially better approach.